price_list.py

```diff
@@ -54,6 +55,17 @@ def compute_discount_rate(self, product, quantity, uom, pattern=None):
         if line:
             return line.discount_rate
 
+    def compute(self, product, quantity, uom, pattern=None):
+        unit_price = super().compute(product, quantity, uom, pattern)
+
+        line = self.get_price_line(product, quantity, product.default_uom)
+        if line and line.formula == '0' and line.base_price_formula:
+            base_price = self.compute_base_price(
+                product, quantity, product.default_uom)
+            if base_price is not None:
```

Comment:
After base_price the discount rate (if there's one) should be computed to obtain the unit price.

Reply:
Done a430205